server/server.mjs

```diff
@@ -13,6 +19,20 @@ tasks.get("/", async (request, response) => {
   response.json(tasks);
 });
 
+tasks.get("/parks", async (request, response) => {
+  console.log(request);
+  const { query } = request;
+  console.log("query: ", query);
+  const result = await axios.get(
+    `https://developer.nps.gov/api/v1/parks?api_key=${process.env.NPS_API_KEY}`,
+    { params: query },
+  );
+
+  response.json(result.data);
+});
+
+//developer.nps.gov/api/v1/parks?parkCode=&parkCode=&stateCode=&sort=&api_key=DfbkVVqO5eM8F7cPXqbJVfOmFEHtfmBXsuktlP48
```

Comment:
A h/u (heads-up) that you have your private key (`api_key`) being exposed to the public here.